R/bench-summ.R

```diff
@@ -0,0 +1,166 @@
+#' Function that provides summary and plots for the benchmark run
+#'
+#' @param .res_tibble formatted bench result tibble with roc and prc columns
+#' roc column: Reciever Operator Curve Results (calculated with yardstick)
+#' prc column: Precision-Recall Curve Results (calculated with yardstick)
+#' @return A summary list with TF coverage, ROC, AUROC, PRAUC, Run time,
+#' ROC plots, and Heatmap plots
+#' @import ggplot2 pheatmap
+#' @importFrom rlang .data
```

Comment:
Not necessary to use, but consider add this prefix to the column names that you want to retrieve to avoid possible inconsistencies.

Reply:
Done.